web-share/idlharness.https.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare IDL test</title>
+    <link rel="help" href="https://wicg.github.io/web-share/">
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+  </head>
+  <body>
+    <script>
+        "use strict";
+        var idl_array = new IdlArray();
+
+        function doTest(idl) {
+          idl_array.add_untested_idls('interface Navigator {};');
+          idl_array.add_idls(idl);
+          idl_array.add_objects({
+            Navigator: ['navigator']
+          });
+          idl_array.test();
+        }
+
+        promise_test(function() {
```

Comment:
I'd love to do that (I write all my code in this style now) but I was afraid that some browsers don't support async/await yet and I don't want the tests to rely on that. Do you know if this is valid in all browsers we care about?